### PR TITLE
feat: add labels with model and app ports

### PR DESCRIPTION
**Pull request by feloy**
### What does this PR do?

Adds labels to the application's pod giving the ports of the model and app, so the Environments manager can know them.
 

### What issues does this PR fix or reference?

As part of #61 

### How to test this PR?

Start an application with ai-studio.yaml containing ports information for model and/or app, and check the labels exist in the pod

### CODE REVIEW

File: packages/backend/src/managers/applicationManager.ts [hunk 1]
```diff
@@ -345,13 +346,22 @@ export class ApplicationManager {
     }
 
     // create new pod
+    const labels = {
+      [LABEL_RECIPE_ID]: recipe.id,
+      [LABEL_MODEL_ID]: model.id,
+    };
+    const modelPorts = images.filter(img => img.modelService).flatMap(img => img.ports);
```
Comment (lstocchi) [inline packages/backend/src/managers/applicationManager.ts:353]: Any reason why you also add the model port? As we do not ping the model anymore to see if it's running we should not expose it in the pod, no?

Comment (feloy) [reply packages/backend/src/managers/applicationManager.ts:353]: In the design made by Mo (https://github.com/projectatomic/ai-studio/issues/235), the port of the model is displayed. I can imagine the developer may want to know the model's port, to make some manual requests on it, for example

Comment (lstocchi) [reply packages/backend/src/managers/applicationManager.ts:353]: maybe let's ask @jeffmaury . BTW looks good to me for the rest 👍

**Review by lstocchi — APPROVED** [review]
Tested, works fine!!

**Comment by jeffmaury** [comment]
2 remarks:

- if there are several ports we keep only the first one, we should store all of them and them ask the user to choose
- what is the added value of having them from labels rather that info from pod ?

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**